be/src/exprs/vectorized/json_functions.h

```diff
@@ -115,24 +126,45 @@ class JsonFunctions {
     DEFINE_VECTORIZED_FN(json_query);
 
     /**
+     * @param: [json_object, json_path]
+     * @paramType: [JsonColumn, BinaryColumn]
+     * @return: BooleanColumn
+     */
+    DEFINE_VECTORIZED_FN(json_exists);
+
+    /**
+     * Build json object from json values
+     * @param: [field_name, field_value, ...]
+     * @paramType: [JsonColumn, JsonColumn, ...]
+     * @return: JsonColumn
+     */
+    DEFINE_VECTORIZED_FN(json_object);
+
+    /**
+     * Build json array from json values
      * @param: [json_object, ...]
      * @paramType: [JsonColumn, ...]
      * @return: JsonColumn
      */
     DEFINE_VECTORIZED_FN(json_array);
 
+    static Status native_json_path_prepare(starrocks_udf::FunctionContext* context,
```

Comment:
Why  the method name need to start with `native`?

Reply:
There's already a function named `json_path_prepare`, for string-based JSON implementation. So the new JSON type is called `Native Json` to differentiate.